src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -394,6 +408,7 @@ export class DonationStartFormComponent implements AfterContentInit, OnDestroy,
               },
               {
                 name: environment.matomoAbTest.variantName,
+                percentage: 15,
```

Comment:
My draft test config from Matomo has this in the embed code. Not ideal to hardcode it but I don't really want to further complicate the env config while unsure if & when we would next do a test that's got 3 variants and non-default traffic allocation.